requirements-dev.txt

```diff
@@ -7,15 +7,17 @@ caproto  # required in order to import ophyd test fixtures
 curio  # required in order to import ophyd test fixtures
 coverage
 flake8
-# TEMPORARY UNTIL https://github.com/NSLS-II/ophyd/pull/682 IS RELEASED
-git+git://github.com/awalter-bnl/ophyd@add-configure
 pytest >=3.9
 sphinx
+suitcase-utils[test_fixtures] >=0.1.0rc2
 # These are dependencies of various sphinx extensions for documentation.
 ipython
 matplotlib
 numpydoc
 sphinx-copybutton
 sphinx_rtd_theme
-suitcase-utils
 trio  # required in order to import ophyd test fixtures
+event-model
```

Comment:
This is redundant and should be removed because `event-model` is already a dependency in `requirements.txt`. These requirements should be the ones that are _only_ needed for running tests or building docs. Putting it in two places means the versions can get out of sync and confuse Future Us.

Reply:
updated